butterknife-sample-lib/build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+  }
+  dependencies {
+    classpath "${GROUP}:butterknife-gradle-plugin:${VERSION_NAME}"
+  }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'butterknife'
```

Comment:
@RClass and plugin are two separate things. RClass is used as a hitn to the processor to find the symbol mapping. It works on apps without using the plugin.

The plugin is just used to create a R with all final values.